lib/cog/queries/command.ex

```diff
@@ -43,6 +54,16 @@ defmodule Cog.Queries.Command do
   end
 
   @doc """
+  Retrieve all information about a command
+  """
+  def complete_command(bundle_name, command_name) do
+    named(bundle_name, command_name)
+    |> with_bundle
+    |> with_rules
+    |> with_options
```

Comment:
I don't think so... they build up a single query, without any interaction with the database at all. `Repo.preload` relies on being passed a model struct (or structs) so you can preload _after_ you've already got an instance of something.

Using the query preload, however, allows you to have composable queries.